Lib/ufo2ft/__init__.py

```diff
@@ -52,17 +55,40 @@ def compileTTF(ufo, outlineCompilerClass=OutlineTTFCompiler,
     *convertCubics* and *cubicConversionError* specify how the conversion from cubic
     to quadratic curves should be handled.
     """
-
     outlineCompiler = outlineCompilerClass(
         ufo, glyphOrder, convertCubics, cubicConversionError)
     otf = outlineCompiler.compile()
 
     featureCompiler = featureCompilerClass(
         ufo, otf, kernWriterClass=kernWriterClass, markWriterClass=markWriterClass,
-        mtiFeaFiles=mtiFeaFiles)
+        mtiFeatures=_getMtiFeatures(ufo, mtiFeaFiles))
     featureCompiler.compile()
 
     postProcessor = PostProcessor(otf, ufo)
     otf = postProcessor.process(useProductionNames)
 
     return otf
+
+
+def _getMtiFeatures(ufo, mtiFeaFiles=None):
+    features = {}
+    prefix = "com.github.googlei18n.ufo2ft.mtiFeatures/"
+    for fileName in ufo.data.fileNames:
+        if fileName.startswith(prefix) and fileName.endswith(".mti"):
+            features[fileName[len(prefix):-4]] = ufo.data[fileName]
+
+    # TODO: Remove the ability to pass MTI feature files as side input
+    # to the compiler. https://github.com/googlei18n/fontmake/issues/289
+    if mtiFeaFiles is not None:
+        log.warn("Future versions of this font compiler will remove "
```

Comment:
for me, we could just drop support for this straight away. I think Noto is the only user of this.